### DISCUSSION TITLE
How to use DarkReader to turn a Dark only websites to light mode?

### DISCUSSION THREAD

**Discussion by oeloo** [darkreader/darkreader#10998] [General]
Hi, I really enjoy Dark mode and thus DarkReader. But when it is sunny, I have to turn my Chrome browser to light mode to be able to read something on my computer screen.
The problem is that some websites only have a Dark mode, for example https://www.macrodroidforum.com/
How can we turn them to Light mode with DarkReader ? (DarkReader is initially done to turn light mode website to dark, so it could be able to do the opposite) Regards

**Comment by ngoclong19** [comment]
The setting is inside Filter -> Mode -> Light. You can select Only for..., so this setting only apply to this specific site.
